enclave/sgx/collateralinfo.c

```diff
@@ -427,17 +583,21 @@ oe_result_t oe_get_sgx_quote_verification_collateral(
 
 done:
     if (result == OE_UNSUPPORTED)
-        OE_TRACE_WARNING(
-            "SGX remote attestation is not enabled. To enable, please add\n\n"
-            "from \"openenclave/edl/sgx/attestation.edl\" import *;\n\n"
-            "in the edl file.\n");
+    {
+        if (quote_type == SGX_QUOTE_TYPE || quote_type == TDX_QUOTE_TYPE)
```

Comment:
This can be combined with above to a single if check. Is there a concern about the message "SGX remote attestation" even for TDX quotes?